count_zeros.js

```diff
@@ -0,0 +1,6 @@
+const CountZeros = (A) => {
+  let zerosCount = 0;
+  A.map((elem) => elem === 0 && zerosCount++);
```

Comment:
`.filter` would have been more appropriate here; and then do the final count of the filtered elements (`.length`)